common/mcu-list.txt

```diff
@@ -1,6 +1,8 @@
-atmega32u4
 at90usb1286
-atmega32u2
+at90usb646
 atmega16u2
+atmega16u4
 atmega328p
 atmega32a
```

Comment:
Would it be possible to change this to:
```suggestion
atmega32
```

In regards to #116

Reply:
The better solution would be to add `atmega32a` as an alias of `m32` in avrdude.conf, which I've now done.